alf/networks/encoding_networks.py

```diff
@@ -69,6 +71,9 @@ def __init__(self,
             conv_layer_params (tuppe[tuple]): a non-empty tuple of
                 tuple (num_filters, kernel_size, strides, padding), where
                 padding is optional
+            use_batch_ensemble (bool): whether use BatchEnsemble Conv layers.
```

Comment:
should comment the input and the output in this case.

Reply:
Updated.